valhalla/midgard/sequence.h

```diff
@@ -590,6 +592,9 @@ struct tar {
     // map the file
     mm.map(tar_file, s.st_size);
 
+    // determine opposite of preferred path separator (needed to update OS-specific path separator)
+    const char opp_sep = filesystem::path::preferred_separator == '/' ? '\\' : '/';
```

Comment:
wow, so tars made on windows contain `\` as the path separators in the filenames!? i guess that kind of makes sense since the container is platform agnostic, but it makes it so downstream applications like ours have to reconcile the source of the tar archive. wild.